test/functional/test-log.js

```diff
@@ -53,12 +53,15 @@ describe('Logging', () => {
         log: logSpy,
       },
       setTimeout: timeoutSpy,
+      reportError: error => error,
     };
+    sandbox.stub(self, 'reportError', error => error);
```

Comment:
Is this necessary? It appears the flaky test uses the fake window object, `win`.